views/js/controller/Delivery/monitoring.js

```diff
@@ -597,12 +597,17 @@ define([
                     return `${moment().format(dateFormat)} to ${moment().add('1', 'd').format(dateFormat)}`;
                 }
 
+                function extractOption(object, option, defaultValue) {
+                    return Object.is(object[option], undefined) ? defaultValue : object[option];
```

Comment:
One small question:
is `Object.is` available in ES5? Because, as I remember, it was still a draft, and introduced only in ES6. And, since, some of the customers may possibly use older sprints (96), which not fully ES6-compliant, - would everything be ok while using it?
@bziondik , could you help with that question?

Reply:
Wouldn't it be better to use smth like `_.isUndefined(object[option]) ? defaultValue : object[option]`?

Reply:
@uncleempty  taoProctoring is migrated on ES2015(ES6) https://oat-sa.atlassian.net/wiki/spaces/NEX/pages/78971530/Migration+to+ES2015
We can use:
`Object` and `Array` new built-in (`Objet.assign`, `Array.from`, `Array.of`, etc.)

Reply:
Yep, this question is mainly about next situation: proctoring is migrated, but I'm not sure at all that we would move DEPP's UAT/prod to more recent sprints than 96 (and migration to ES6 was later), at least it wasn't discussed yet. That's why the question was raised.

Reply:
@jsconan what would you suggest?

Reply:
For a backport, indeed it could be an issue, as we won't be able to transpile it.
However, for new development this is totally legit since the extension is turned into ES6 compatible (`babel` flag in the `bundle.js`)